packages/core/module-plugin/test/mock-api/api.js

```diff
@@ -1,5 +1,5 @@
-const { get } = require('@friggframework/assertions');
-const { OAuth2Requester } = require('@friggframework/module-plugin');
+const { get } = require('../../assertions');
```

Comment:
Does this matter if we use local reference vs the package? (Wondering if in some cases it would be better to reference the package/workspace, vs the path, since pathing may change as we iterate)

Reply:
I know that it matters in some cases because it can create circular imports and such. I think if it's in the same package it should use the local reference, but maybe I'm wrong on that